conans/assets/templates/new_v2_bazel.py

```diff
@@ -1,3 +1,5 @@
+import platform
```

Comment:
As this is Conan 1.x, I recommend opening an issue referencing your comment and pointing to Conan 2.x milestone. I think it could be too much for this PR (new definition, parameter in `conan new` templates, tests, etc.)